zng/docs/spec.md

```diff
@@ -648,7 +647,8 @@ and references the types accordingly in three values;
 1:[hello;world;]
 0:this is a semicolon: \x3b;
 ```
-which represents a stream of the following three values:
+which represents a stream of the following three values (expressed as
+pseudocode):
```

Comment:
It's always bothered me a little that what's shown below looks like structured syntax but we never really say what "it" is. If we wanted to reference some kind of lingua franca instead, perhaps we could write the values as JSON & say that's what it is. Opinions?

Reply:
I would just drop the entire section 4.4, seeing as much  it is not currently implemented. If anything I'd say the zng section needs examples more than the tzng section, as tzng is easier to understand from the spec and to manually experiment with.

Reply:
I've talked to @henridf offline about this one. We're both in agreement that it feels a little hazardous to leave so many non-working examples. The line I added to the alpha notice is probably not enough for a first-time reader to know how to stay inside the lines if they start experimenting with `zq` while they're making their way through the spec. I've tried to add a touch more guidance via the Zeek example I moved into this section from the now-deleted "ZNG Data Model" section" and describing it as a "common case", but that still may not be enough. At the risk of being too heavy with disclaimers, maybe one more alpha-style notice would be justified at the start of this Examples section just to call attention to the lag in the implementation and call their attention to the Zeek example as the most representative one. Anyone have opinions?